### ISSUE TITLE
Call to a member function hasPermissionTo() on null

### ISSUE THREAD

**Issue by fogs** [zubroide/gitpab#21]
While trying to build the docker containers, I am getting

```
app_1  |  DONE  Compiled successfully in 14779ms08:14:30
app_1  | 
app_1  |                                                                                                     Asset     Size  Chunks                    Chunk Names
app_1  |   fonts/vendor/bootstrap-sass/bootstrap/glyphicons-halflings-regular.eot?f4769f9bdb7466be65088239c12046d1  20.1 kB          [emitted]         
app_1  | fonts/vendor/bootstrap-sass/bootstrap/glyphicons-halflings-regular.woff2?448c34a56d699c29117adc64c43affeb    18 kB          [emitted]         
app_1  |  fonts/vendor/bootstrap-sass/bootstrap/glyphicons-halflings-regular.woff?fa2772327f55d8198301fdb8bcfc8158  23.4 kB          [emitted]         
app_1  |   fonts/vendor/bootstrap-sass/bootstrap/glyphicons-halflings-regular.ttf?e18bbf611f2a2e43afc071aa2f4e1512  45.4 kB          [emitted]         
app_1  |   fonts/vendor/bootstrap-sass/bootstrap/glyphicons-halflings-regular.svg?89889688147bd7575d6327160d64e760   109 kB          [emitted]         
app_1  |                                                                                                /js/app.js   523 kB       0  [emitted]  [big]  /js/app
app_1  |                                                                                              /css/app.css   127 kB       0  [emitted]         /js/app
app_1  |                                                                                /js/moment-with-locales.js   232 kB          [emitted]         
app_1  | Configuring the database
app_1  | Nothing to migrate.
app_1  | Seeding: RolePermissionSeeder
app_1  | 
app_1  | In EloquentUserService.php line 27:
app_1  |                                                        
app_1  |   Call to a member function hasPermissionTo() on null  
app_1  |                                                        
app_1  | 
docker_app_1 exited with code 1
```

How to reproduce:
- clone repo from acbd4c3e5613a40ab73e573f475edadb8e7bbd15 (current master)
- cd gitpab/docker
- export host="https://projects.my.domain/" \
    && export token="xyz" \
    && export projects="42" \
    && docker-compose up --build app

**Comment by mnvx** [comment]
Fixed in #20. Try again please.

**Comment by fogs** [comment]
Wow, fastest response & bugfix I have ever gotton on any github project. Kudos to you, my friend!
